src/TrackingAction.cpp

```diff
@@ -59,6 +59,12 @@ void TrackingAction::PreUserTrackingAction(const G4Track* track)
         )
     );
 
+    //G4cout << "Track info: " << G4endl
+    //       << "  Track ID:       " << track->GetTrackID() << G4endl
+    //       << "  Track PDG:      " << track->GetDefinition()->GetPDGEncoding() << G4endl
+    //       << "  Track Momentum: " << track->GetMomentum().x() << ", " << track->GetMomentum().y() << ", " << track->GetMomentum().z() << G4endl
+    //       << "  Track Energy:   " << track->GetTotalEnergy() << G4endl; 
+
```

Comment:
Don't leave commented-out code in the repository.

Reply:
fixing in bugfix/v3.0.0_revisions branch